mws/sitesmanagement/models.py

```diff
@@ -590,6 +590,11 @@ class Vhost(models.Model):
     WEBAPP_CHOICES = (
         ('wordpress', 'Wordpress'),
     )
+
+    ALL_NAMES = ['denied', 'requested']
+    GLOBAL_NAMES = ['global', 'external', 'special']
+    PRIVATE_AND_GLOBAL_NAMES = GLOBAL_NAMES.extend(['accepted', 'private'])
```

Comment:
I think this line actually modifies GLOBAL_NAMES.
```suggestion
    PRIVATE_AND_GLOBAL_NAMES = GLOBAL_NAMES + ['accepted', 'private']
```

Reply:
it does, doesn't it. fixed.